admin_guide/sdn_troubleshooting.adoc

```diff
@@ -495,6 +495,54 @@ normal operation, so do not be surprised to see entries there.
 
 Check external firewalls, if any, allow traffic to the target address (this is site-dependent, and beyond the purview of this guide).
 
+== Debugging Virtual Networking
+
+=== Builds on a Virtual Network are Failing
+
+If you are installing OpenShift using a virtual network (for example,
+OpenStack), and a build is failing, the node maximum transmission unit (MTU)
+might not be compatible with the network MTU (for example, eth0).
```

Comment:
My understanding is that `eth0` is an example of an *interface*.  Suggest either using a different example, or dropping it.